src/main/java/easton/Easton.java

```diff
@@ -0,0 +1,263 @@
+package easton;
+
+import easton.model.Deadline;
+import easton.model.Event;
+import easton.model.Task;
+import easton.model.ToDo;
+import easton.view.Ui;
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class Easton {
```

Comment:
Perhaps you could include javadoc header comments for classes and methods so that readers can better understand your code.

Reply:
Hi @weijianwong! yes I was late in pushing the javadoc comments, I have included it for all my public classes and methods.